src/syscheckd/src/main.c

```diff
@@ -17,6 +17,9 @@
 
 #ifndef WIN32
 
+#define Q(x) #x
```

Comment:
Why was this definition added?

Reply:
this is to add quotes to the defined value on the command line.